MCSync/frmConfig.cs

```diff
@@ -90,6 +62,11 @@ private void frmConfig_Load(object sender, EventArgs e)
 
             rbClient.Checked = syncMode == SyncMode.Client;
             rbServer.Checked = syncMode == SyncMode.Server;
+
+            overrideFoldersCheckBox.Checked = string.IsNullOrEmpty(configValues.GetOrDefault("minecraftDirectory").Cast<string>());
+            overrideFoldersCheckBox_CheckedChanged(this, EventArgs.Empty);
```

Comment:
Pass in `overrideFoldersCheckBox` instead of `this` for the first parameter

Reply:
Fixed :3